src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnection.cs

```diff
@@ -2023,7 +2023,7 @@ public async ValueTask DrainResponseAsync(HttpResponseMessage response, Cancella
 
             if (_connectionClose)
             {
-                throw new HttpRequestException(SR.net_http_authconnectionfailure);
+                throw new HttpRequestException(SR.net_http_authconnectionfailure, httpRequestError: HttpRequestError.UserAuthenticationError);
```

Comment:
Is `UserAuthenticationError` meant to encapsulate only "wrong username/password" or "something went wrong with auth"?
The description sounded to me like the former, but this looks like the latter.

(same for the exception below)

Reply:
Update the docs for `UserAuthenticationError`.